src/slack.js

```diff
@@ -269,6 +269,15 @@ async function handleCSVUpload(event, bot, say) {
     text: `*Invoices summary*\n${formattedInvoices}`,
   })
 
+  if (!bot.sendOnSlack) {
```

Comment:
I contacted the Finance department to confirm we can really hide the buttons. facts:
- there are 5 production channels and 4 of them have `sendOnSlack` disabled
- the `Send invoices` button also generates, uploads, and shares the PDFs on Google Drive - I'm not sure whether nobody uses this functionality

Reply:
okay, so in order to preserve uploading to Google Drive, I only updated the texts and "sending" button labels to reflect whether `sendOnSlack` is enabled or not.